### DISCUSSION TITLE
Material UI Select onChange Event

### DISCUSSION THREAD

**Discussion by GabrielGomesBarbosa** [react-hook-form/react-hook-form#3169] [Q&A] [answered]
Hi @bluebill1049 , i was search by an example with `react-hook-form` and `material-ui`, i have a select and i'm using it with `<Controller />`  and `as` prop but the onChange event is not triggered, i read the documentation and in v6 was included `render={}` prop, its work fine to get value, but when i use to edit with `defaultValues` the value was not set. Can you give a example to use MUI Select with onChange ? it's can be via `render` or `as` prop.

Thanks

**Comment by GabrielGomesBarbosa** [comment]
When i use `Controller` with `render` prop without `defaultValue={something}` i receive:

![image](https://user-images.githubusercontent.com/37483241/95907949-b50ea200-0d72-11eb-9d7d-61c579b18893.png)

when i set `defaultValue` its dont work.

**Reply by bluebill1049** [reply]
CSB plz

**Reply by GabrielGomesBarbosa** [reply]
I Will provide it as soon as possible, im read the docs again before build the sand box!

**Reply by kaisiant** [reply]
Perhaps you need this. https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r?file=/src/Mui.js

**Comment by GabrielGomesBarbosa** [comment, marked as answer]
Thanks @kaisian97 was that.
